packages/array-max/src/index.js

```diff
@@ -3,9 +3,13 @@ import isArray from 'is-any-array';
 /**
  * Computes the maximum of the given values
  * @param {Array<number>} input
+ * @param {int} [options.from] - Start index (inclusive) for the slice within which we look for the max
```

Comment:
```suggestion
 * @param {number} [options.from] - Start index (inclusive) for the slice within which we look for the max
```

`int` doesn't exist in JS/JSDoc